src/nft_contract/src/main.nr

```diff
@@ -9,34 +9,31 @@ pub contract NFT {
     use crate::types::nft_note::{NFTNote, PartialNFTNote};
     // aztec library
     use aztec::{
-        encrypted_logs::log_assembly_strategies::default_aes128::note::encode_and_encrypt_note,
+        context::PrivateContext,
         macros::{
-            functions::{initializer, internal, private, public, view},
+            functions::{initializer, internal, private, public, utility, view},
             storage::storage,
         },
         note::{constants::MAX_NOTES_PER_PAGE, note_interface::NoteProperties},
         prelude::{
-            AztecAddress, Map, NoteGetterOptions, NoteViewerOptions, PrivateContext, PrivateSet,
+            AztecAddress, Map, NoteGetterOptions, NoteViewerOptions, PrivateSet,
             PublicContext, PublicImmutable, PublicMutable,
         },
         protocol_types::traits::Serialize,
         utils::comparison::Comparator,
     };
     // authorized witness library
     use authwit::auth::{
-        assert_current_call_valid_authwit, assert_current_call_valid_authwit_public,
-        compute_authwit_nullifier,
+        assert_current_call_valid_authwit, assert_current_call_valid_authwit_public
     };
     // compression library
     use compressed_string::FieldCompressedString;
-    // aztec protocol types
-    use aztec::protocol_types::traits::ToField;
-    // std library
-    use std::meta::derive;
+    // encryption library
+    use aztec::messages::logs::note::encode_and_encrypt_note;
```

Comment:
we can move it to the `aztec::{` section, it was separated before when working with macros to be declared on each contract but not pursuing that roadmap for now